### PR TITLE
Fix for #692. Iterate up to a parent when matching subobject.

### PR DESCRIPTION
**Pull request by bnolan**
Changes proposed:
- Iterate up object tree to the element that is attached to the dom.

### CODE REVIEW

File: src/components/raycaster.js [hunk 1]
```diff
@@ -66,11 +66,26 @@ module.exports.Component = registerComponent('raycaster', {
...
+      }
+
       // If the intersected object is the cursor itself
       // or the object is further than the max distance
-      if (intersectedObj.object.el === undefined) { continue; }
-      if (intersectedObj.object.el === cursorEl) { continue; }
-      if (!intersectedObj.object.visible) { continue; }
+
+      if (intersectedObj.object.el === undefined) {
+        continue;
```
Comment (dmarcos) [inline src/components/raycaster.js]: Let's keep these as one liners

File: src/components/raycaster.js [hunk 2]
```diff
@@ -66,11 +66,26 @@ module.exports.Component = registerComponent('raycaster', {
     var intersectedObjs = this.intersect(scene.children);
     for (var i = 0; i < intersectedObjs.length; ++i) {
       intersectedObj = intersectedObjs[i];
+
+      while (
+        (intersectedObj.object.parent) &&
```
Comment (ngokevin) [inline src/components/raycaster.js]: we can put this if statement on one line without the parens

and comment the while loop ```//  Iterate up to parent when matching subobject.```

**Comment by dmarcos** [comment]
This is almost ready to go. @bnolan a couple of quick changes and we can merge

**Comment by bnolan** [comment]
Ok sweet, made those changes! :+1:

**Comment by dmarcos** [comment]
Thanks!

**Pull request merged**